lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -75,8 +81,11 @@ defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
     end
   end
 
-  defp relevant_alert?(alert) do
-    relevant_effect?(alert) and informs_one_facility?(alert)
+  # All alerts at home station are relevant but only elevators without redundancies are
+  # relevant at other stations.
+  defp relevant_alert?(alert, home_stop_id) do
+    relevant_effect?(alert) and informs_one_facility?(alert) and
+      (Alert.informs_stop_id?(alert, home_stop_id) or not has_nearby_redundancy?(alert))
```

Comment:
When an alert informs a facility, does it also always inform the facility's parent station? This isn't necessarily true of GTFS in general but I'd believe that it's a guarantee of our alerts implementation.

Reply:
From what I can tell, yes. I think it always includes the parent + every child stop not flagged for exclusion in the `properties` field for the facility.